### PR TITLE
feat: Add autoCommit option

### PR DESCRIPTION
**Pull request by galvez**
@dead-horse think I got it right now :) re: #138

### CODE REVIEW

File: lib/context.js [hunk 1]
```diff
@@ -197,6 +197,14 @@ class ContextSession {
     debug('create session with val: %j externalKey: %s', val, externalKey);
     if (this.store) this.externalKey = externalKey || this.opts.genid();
     this.session = new Session(this.ctx, val);
+    if (!this.opts.autoCommit) {
```
Comment (dead-horse) [inline lib/context.js]: I'd like to move this method here https://github.com/koajs/session/blob/master/lib/session.js . and if we named the method as `manuallyCommit`,I think it's ok to always add this method even `autodCommit` was set to true.

Comment (galvez) [reply lib/context.js]: @dead-horse `Session` has no reference to ContextSession so I had to pass it.

Comment (galvez) [reply lib/context.js]: @dead-horse Updated!

**Comment by galvez** [comment]
@dead-horse Any additional feedback? Personally, I'd prefer not to change `Session`'s constructor just to have `manuallyCommit()` defined in there, but I'll let you be the judge of that.

**Comment by dead-horse** [comment]
LGTM

**Comment by dead-horse** [comment]
5.9.0 released, thanks!

**Pull request merged**